inc/question.class.php

```diff
@@ -259,33 +259,24 @@ public function getDesignHtml() : string {
     * Get the HTML to display the question for a requester
     * @param string  $domain  Translation domain of the form
     * @param boolean $canEdit Can the requester edit the field of the question ?
-    * @param array   $value   Values all fields of the form
+    * @param PluginFormcreatorFormAnswer $value   Values all fields of the form
     * @param bool $isVisible is the question visible by default ?
     *
     * @return string
     */
-   public function getRenderedHtml($domain, $canEdit = true, $value = [], $isVisible = true): string {
+   public function getRenderedHtml($domain, $canEdit = true, ?PluginFormcreatorFormAnswer $form_answer = null, $isVisible = true): string {
```

Comment:
replace the array of values with the instance of Form Answer. From this instance, we can get the values